2006-10-02  Jakub Jelinek  <jakub@redhat.com>

	[BZ #3291]
	* sysdeps/unix/sysv/linux/sparc/sparc64/pause.c: Include
	errno.h, signal.h, unistd.h and sysdep-cancel.h.
	(__sigprocmask): Define.

===================================================================
RCS file: /cvs/glibc/libc/sysdeps/unix/sysv/linux/sparc/sparc64/pause.c,v
retrieving revision 1.4
retrieving revision 1.5
diff -u -r1.4 -r1.5
--- libc/sysdeps/unix/sysv/linux/sparc/sparc64/pause.c	2006/08/15 05:27:23	1.4
+++ libc/sysdeps/unix/sysv/linux/sparc/sparc64/pause.c	2006/10/02 18:40:39	1.5
@@ -1 +1,9 @@
+#include <errno.h>
+#include <signal.h>
+#include <unistd.h>
+#include <sysdep-cancel.h>
+
+#define __sigprocmask(how, set, oset) \
+  INLINE_SYSCALL (rt_sigprocmask, 4, how, set, oset, _NSIG / 8)
+
 #include <sysdeps/posix/pause.c>
